content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -267,3 +273,91 @@ $ kubectl delete -f $HOME/istio-remote.yaml
 ```command
 $ helm delete --purge istio-remote
 ```
+
+## Stable access to istio control plane services
```

Comment:
Wasn't load balancer a security issue?  Integrating this into the chart is easy as a flag.  Expecting operators to change the manifests is asking for trouble.

I thought we had agreed to use a unique gateway?  the gateway code has been modified to allow any number of gateways to be added for any purpose.  See:

https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L139-L145

We can add a gateway for this purpose easily and I'd be willing to add the gateway.  The challenge is what to do *with* the gateway once added.

cc @costinm

Reply:
@sdake has you seen my follow-up email in the istio-dev multi cluster LB thread? Like you said, adding a gateway workload (if that's what you mean) is easy, but we can't create a complete solution for the control plane around it for remote access yet.

I'm not aware that using LB is a security issue. Can you clarify?

IIRC Guang had tested the LB with the charts earlier on.